modules/ROOT/pages/install-port-reqs.adoc

```diff
@@ -5,6 +5,13 @@ endif::[]
 
 Runtime Fabric requires specific network and port configuration for installation and operation.
 
+== Bandwidth Requirements
+
+A high bandwidth network is required, with a minimum of 1 GBps connectivity in the internal network running the RTF nodes. 
```

Comment:
consider rewording -- suggestion: "..between each node operating Runtime Fabric."

Reply:
We should also add a requirement with outbound connectivity to the Internet: 100 mbps.

Reply:
Got it - thank you!